cmds/storaged/main.go

```diff
@@ -65,6 +66,12 @@ func main() {
 		log.Info().Msg("shutting down")
 	})
 
+	go func() {
+		if err := http.ListenAndServe(":8080", http.DefaultServeMux); err != nil {
```

Comment:
What is this?

Reply:
https://github.com/threefoldtech/zos/pull/931#issuecomment-691037449